test/basic-querying.test.js

```diff
@@ -593,6 +613,19 @@ describe('basic-querying', function() {
 
     describe('geo queries', function() {
       describe('near filter', function() {
+        it('supports a near query with null condition', function(done) {
```

Comment:
could you move this test right after the 'supports a basic "near" query' test please?